### DISCUSSION TITLE
Codewars sign in error

### DISCUSSION THREAD

**Discussion by Jorudith** [codewars/codewars.com#2490] [Q&A]
Good day everyone

I recently ran into a problem of not being able to access my account on codewars that is linked to my github account.

The error says "Unknown GitHub user @Jorudith(My name). Sign up or link your account first.

I've linked the account and I've received a confirmation email and did some projects but when I didn't log in for a while. The error above happened and I don't know what to do with it

I hope someone knows the solution to this

Thank you

**Comment by kazk** [comment]
Your Codewars account https://www.codewars.com/users/Jorudith doesn't have GitHub account associated with it. Unfortunately, it's difficult to find what happened.

Looks like you did this already, but for anyone having this issue, you can request a password reset email to log in. GitHub account can be linked from Account Settings.

**Reply by Jorudith** [reply]
Yea, After this incident, I tried to create another account using the previous email that I used and it says that it's in use, so I unlinked my account but that didn't do anything, ergo this post. 

Thank you for the information though.

**Comment by prince24j** [comment]
Currently I am facing similar issue, is there any legit way to fix this issue?  


![a](https://user-images.githubusercontent.com/59372915/211769267-1d3b315f-2847-458c-8eb4-8ecbf47a06fc.png)

**Comment by Mohamed-Yacine-Y** [comment]
I think you should go back to the landing page, then click on "join" after that click "Link Your GitHub Account" .
![Screenshot 2023-07-06 130905](https://github.com/codewars/codewars.com/assets/110281284/cebc826d-83a0-44b9-8cf0-b6240ff48279)

**Reply by Hendrik2319** [reply]
Sorry, that doesn't work.
I get the same result as @prince24j, when I use the "log in" button on landing page an then "sign in with github".
If I click on "join" at landing page and then "Link your github account" as you show in your screenshot, then I get an "internal server error".
![internal server error](https://github.com/codewars/codewars.com/assets/135076120/49099aea-0b78-4893-a89e-c2ab3137687a)

**Reply by Hendrik2319** [reply]
If I want to create a new account with an email address, username and password, I get email "is invalid".
When I use the same email address that I'm using in my github account, I get email "is invalid" too.
Both addresses end with "@nurfuerspam.de".
And both addresses are working.
Is the email address check buggy?

**Reply by hobovsky** [reply]
Sign-up process does not accept emails from service providers of discardable email inboxes. It is possible that email address from domain `nurfuerspam.de` is simply rejected.

**Reply by Hendrik2319** [reply]
An address good enough for GitHub is not good enough for Codewars?

**Reply by alex1yaremchuk** [reply]
> k you should go back to the landing page, then

Thank you, it worked for me! 

I see that i mixed up 'sign up' and 'sign in' steps.

**Comment by katarakta2** [comment]
if this is still relevant, then clearing cookies and clearing browser history just helped me.

**Comment by BLAST-Z** [comment]
yea it happened with me too and i used my second accound lol.
